src/server/cluster/cluster_family.cc

```diff
@@ -502,10 +502,18 @@ void ClusterFamily::DflyClusterConfig(CmdArgList args, ConnectionContext* cntx)
     before = tl_cluster_config->GetOwnedSlots();
   }
 
-  auto cb = [&](util::ProactorBase* pb) { tl_cluster_config = new_config; };
+  DispatchTracker tracker{server_family_->GetListeners(), cntx->conn()};
+  auto cb = [&tracker, &new_config](util::ProactorBase* pb) {
+    tl_cluster_config = new_config;
+    tracker.TrackOnThread();
+  };
```

Comment:
So we can now pause writes _when_ replacing the cluster config, to actually return an error without applying. wdyt?

Reply:
As in

```kotlin
run {
  paused = true
  tracker.track_on_thread()
}

if not tracker.wait( 1s ): 
  return "Can't replace cluster config with constant ops running"

# no writes
run {
  tl_config = new_config
}
```

but why should we pause under heavy load without long running ops 😞 ?

One more option would be using a global tx, but that doesn't prevent write commands from still being scheduled